fonts/Makefile.am

```diff
@@ -120,6 +120,10 @@ dist_font_DATA = \
 	Skranji-Regular.ttf \
 	Sniglet-ExtraBold.ttf \
 	Sniglet-Regular.ttf \
+	SutonnyMJ.ttf \
+	SutonnyMJ-Bold.ttf \
+	SutonnyMJ-BoldItalic.ttf \
+	SutonnyMJ-Italic.ttf \
```

Comment:
Are these all the fonts we have? If there are more, we should add them anyways.

Also, we must add the font license!